src/Skeletor/Frameworks/Framework.php

```diff
@@ -86,11 +87,11 @@ public function setVersion(string $version)
      */
     public function getPath(string $path)
     {
-        if(array_key_exists($path, $this->paths)){
-            return $this->paths[$path];
+        if(array_key_exists($path, $this->paths) === false){
```

Comment:
@ThijsBouwes @titopixelfusion is right, we we do it in other places already so then it's better to stick to that. So I'm happy for it to be changed to `!array_key_exists(...)`